### PR TITLE
EDU-997: add UI config options

### PR DESCRIPTION
**Pull request by DJSanti**
## What does this PR do?
This PR adds new Web UI configuration options per the latest CLI release.

## Notes to reviewers
Edits were made to web-ui-configuration. Make it known if additional variables need to be added to server-config and other related reference files.

### CODE REVIEW

File: docs-src/references/web-ui-configuration.md [hunk 1]
```diff
@@ -107,55 +94,22 @@ When enabled—that is, when set to `true`—a notification appears in the UI wh
...
-        organization: org_xxxxxxxxxxxx
-        invitation:
-```
-
-## tls
+## refreshInterval
 
-Transport Layer Security (TLS) configuration.
+How often the configuration UI Server reads the configuration file for new values.
+Currently, only [tls](#tls) configuration values are propagated during a refresh.
```
Comment (napcs) [inline docs-src/references/web-ui-configuration.md:100]: capitalize TLS?

File: docs-src/references/web-ui-configuration.md [hunk 2]
```diff
@@ -204,3 +158,89 @@ Prevents the user from resetting Workflows from the Web UI.
...
+  cookieInsecure: false
+  allowOrigins:
+    - http://localhost:3000 # used at development by https://github.com/temporalio/ui
+```
+
+## tls
+
+Transport Layer Security (TLS) configuration.
+
+```yaml
```
Comment (napcs) [inline docs-src/references/web-ui-configuration.md:194]: should this example have some default example values like the auth configuration does?

File: docs-src/references/web-ui-configuration.md [hunk 3]
```diff
@@ -204,3 +158,89 @@ Prevents the user from resetting Workflows from the Web UI.
...
+  caData:
+  certData:
+  keyData:
+  enableHostVerification: false
+  serverName:
+```
+
+## auth
+
+Auth configuration.
```
Comment (napcs) [inline docs-src/references/web-ui-configuration.md:208]: Do we have more information about this somewhere in docs? I'd like to know what these fields in the YAML mean. What are the possible providers people can use, etc.

File: docs-src/references/web-ui-configuration.md [hunk 4]
```diff
@@ -14,62 +14,58 @@ An example development.yaml file can be found in the [temporalio/ui-server repo]
...
 
-## enableUi
+## temporalGrpcAddress
 
-Enables the browser UI.
-This configuration can be set dynamically with the [TEMPORAL_UI_ENABLED](/references/web-ui-environment-variables#temporaluienabled) environment variable.
-If disabled—that is, set to `false`—the UI server APIs remain available.
+The frontend address for the Temporal Cluster.
+
+The default address is localhost (127.0.0.1:7233).
```
Comment (napcs) [inline docs-src/references/web-ui-configuration.md:21]: should probably state address and port, right?

File: docs-src/references/web-ui-configuration.md (hunk 3 above)
Comment (DJSanti) [reply docs-src/references/web-ui-configuration.md:208]: Not in docs. I can ask the engineers.

File: docs-src/references/web-ui-configuration.md (hunk 2 above)
Comment (DJSanti) [reply docs-src/references/web-ui-configuration.md:194]: The defaults are all empty strings.

**Comment by DJSanti** [comment]
Converting to a draft until I'm provided some information from those that work on UI.

File: docs-src/references/web-ui-configuration.md (hunk 2 above)
Comment (DJSanti) [reply docs-src/references/web-ui-configuration.md:194]: Finally got a reply. Fixing this one right up.

**Review by napcs — APPROVED** [review]

**Pull request merged**